vm/src/protocol/number.rs

```diff
@@ -11,8 +11,10 @@ use crate::{
     VirtualMachine,
 };
 
+pub type PyNumberInquiryFunc = fn(PyNumber, &VirtualMachine) -> PyResult<bool>;
```

Comment:
Is this a special form of unary func?
```suggestion
pub type PyNumberInquiryFunc = PyNumberUnaryFunc<bool>;
```

Reply:
Yes its right, but just `PyNumberUnaryFunc<bool>` seems ok at this point.